api/src/klicenses/resolvers/klicense_mutations.ts

```diff
@@ -6,9 +6,15 @@ import { KLicense } from "../klicense";
 export function KotsLicenseMutations(stores: Stores) {
   return {
     async syncAppLicense(root: any, args: any, context: Context): Promise<KLicense> {
-      const { appSlug, airgapLicense } = args;
+      const { appSlug } = args;
+      let airgapLicense: string = args.airgapLicense;
```

Comment:
@salahalsaleh1993 is this correct/good way to force this to be a string?

Reply:
yes

Reply:
but it is already enforced in here https://github.com/replicatedhq/kotsadm/blob/master/api/src/schema/mutation.ts#L26

Reply:
nice, ok!